main/tests/server/src/com/google/refine/operations/column/ColumnReorderOperationTests.java

```diff
@@ -71,15 +75,34 @@ public void createProject() {
 
     @Test
     public void serializeColumnReorderOperation() {
-        AbstractOperation op = new ColumnReorderOperation(Arrays.asList("b", "c", "a"));
+        AbstractOperation op = new ColumnReorderOperation(Arrays.asList("b", "c", "a"), false);
         TestUtils.isSerializedTo(op, "{\"op\":\"core/column-reorder\","
                 + "\"description\":" + new TextNode(OperationDescription.column_reorder_brief()).toString() + ","
+                + "\"isPureReorder\": false,"
                 + "\"columnNames\":[\"b\",\"c\",\"a\"]}");
     }
 
+    @Test
+    public void deserializeLegacyFormat() throws JsonMappingException, JsonProcessingException {
+        ColumnReorderOperation op = ParsingUtilities.mapper.readValue("{\"op\":\"core/column-reorder\","
+                + "\"columnNames\":[\"b\",\"c\",\"a\"]}", ColumnReorderOperation.class);
+        assertEquals(op._columnNames, List.of("b", "c", "a"));
+        assertEquals(op._isPureReorder, false);
```

Comment:
@SoryRawyer The test above shows the backwards-compatibility for this change. The old serialization format of this operation (without the `isPureReorder` field) will still be understood as before, as an operation that removes any columns that are not mentioned in the `columnNames` list.

This means that if users execute recipes generated from a previous version of OpenRefine with the new version, they should still get the same results.